### ISSUE TITLE
how do you generate call graph?

### ISSUE THREAD

**Issue by ramsey-coding** [anonymoussubmission9/anonymous-submission#1]
how do you generate call graph? They are not available in the replication package.

**Comment by token955** [comment]
> how do you generate call graph? They are not available in the replication package.

Yes, they are like grace, this part of the work is not public, only the pre-processed results.

**Comment by anonymoussubmission9** [comment]
The call graph was generated using this tool: https://github.com/gousiosg/java-callgraph
